lib/api/settings.rb

```diff
@@ -5,4 +5,9 @@ module Api
     o.add_source!(Rails.root.join("config/api.yml").to_s)
     o.load!
   end
+
+  def self.model_to_collection(klass)
```

Comment:
If this is a method on `Api` perhaps it should live under lib/api.rb? Unless you could instance_eval `Settings` to add this convience method to it?